erpnext/accounts/doctype/gl_entry/gl_entry.py

```diff
@@ -462,4 +462,8 @@ def rename_temporarily_named_docs(doctype):
 				f"UPDATE `tab{doctype}` SET name = %s, to_rename = 0, modified = %s where name = %s",
 				(newname, now(), oldname),
 			)
+
+			for hook in frappe.get_hooks("on_gle_sle_rename"):
+				frappe.get_attr(hook)(newname, oldname)
```

Comment:
1. Use `frappe.call` instead of `frappe.get_attr` + `()`, `frappe.call()` makes the hooks flexible for future extensions.
2. Specify which doctype is being renamed too. (You might need to recognize SLE/GLE difference)

Reply:
Done number 1.
But number 2, here is what i try to do is to rename the link field "GL Entry" which was in Sales/Purchase Tax Invoice.
https://github.com/ecosoft-frappe/erpnext_thailand/blob/8e8990575db8204b60dbe36912d7fc244fba13b1/erpnext_thailand/custom/gl_entry.py

In fact, it will never be Stock Ledger Entry.

Or should we be more sepecific and have 2 hooks in rename_temporarily_named_docs()?
* on_gle_rename (I will only use this one)
* on_sle_rename

Reply:
Yeah two separate hooks are better.

Someone else might have use case for sle rename, so might as well add both for consistency.

Reply:
@ankush thanks and done.